src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -104,6 +105,6 @@ public Command intake() {
             spinnyClaw.stop(),
             elevator.setLevel(ElevatorSubsystem.PRE_INTAKE),
             stow())
-        .withName("Intake");
+        .withName("intake");
```

Comment:
I'd recommend reverting this to avoid a future merge conflict since #43 changes this name to something else,